packages/create-plugin/src/commands/update.command.ts

```diff
@@ -1,81 +1,80 @@
-import { TEXT, UDPATE_CONFIG } from '../constants.js';
-import { compileTemplateFiles, getTemplateData } from '../utils/utils.templates.js';
-import { confirmPrompt, selectPrompt, printMessage, printSuccessMessage } from '../utils/utils.console.js';
-import {
-  updatePackageJson,
-  hasNpmDependenciesToUpdate,
-  getPackageJsonUpdatesAsText,
-  updateNpmScripts,
-  writePackageManagerInPackageJson,
-} from '../utils/utils.npm.js';
-import { getPackageManagerWithFallback } from '../utils/utils.packageManager.js';
+import chalk from 'chalk';
+import { EXTRA_TEMPLATE_VARIABLES } from '../constants.js';
+import { updatePluginBuildConfig, isPluginDirectory } from '../utils/utils.plugin.js';
+import { getRuntimeVersion } from '../utils/utils.version.js';
+import { printRedBox, printBlueBox } from '../utils/utils.console.js';
+import { updatePackageJson, updateNpmScripts } from '../utils/utils.npm.js';
+import { isGitDirectory, isGitDirectoryClean } from '../utils/utils.git.js';
 
 export const update = async () => {
   try {
-    // 0. Warning
-    // ----------
-    printMessage(TEXT.updateCommandWarning);
+    if (!(await isGitDirectory())) {
```

Comment:
I see the message talks about a `--force` flag but there's no logic to handle that

Reply:
Fair point! (Added a note to the PR description about not existing yet, but I agree it's confusing, I will fix it.)

Reply:
Resolved in https://github.com/grafana/plugin-tools/pull/702/commits/e3720a6d75fb873ba700efd733be38dcc683e4cf.